assets/src/scss/new-identity/style.scss

```diff
@@ -1,6 +1,7 @@
 @import "../base/tokens";
 @import "../base/variables";
 @import "../base/mixins";
+@import "./fonts";
```

Comment:
This implementation is not part of this ticket. It's added within https://github.com/greenpeace/planet4-master-theme/pull/2008